setup.py

```diff
@@ -23,7 +23,7 @@
 
 install_requires = [
     'six>=1.7.2',
-    'isbnlib>=3.5.6',
+    'isbnid==0.3.4',
```

Comment:
What about ``isbnid>= 0.3.4``? If that doesn't work, please go with ``isbnid>=0.4.2`` and disable the ``lowest`` build matrix. It's not good to pin a requirement in setup.py as it can cause a lot of installation issues especially if another library pins the same requirement - it's far better to  simply leave the version unspecified than pinning it unless absolutely necessary. See [abstract vs concrete requirements](https://caremad.io/2013/07/setup-vs-requirement/) for a good read about it.